src/@types/express-http-proxy.d.ts

```diff
@@ -0,0 +1,22 @@
+interface Options {
```

Comment:
I will try to move these upstream into https://github.com/DefinitelyTyped/DefinitelyTyped/